enigma-principal/enclave/src/keys_keeper_t/mod.rs

```diff
@@ -52,7 +52,7 @@ fn get_state_keys(keys_map: &mut HashMap<ContractAddress, StateKey>,
                 println!("State key not found in cache, fetching sealed document.");
```

Comment:
Shouldn't this print move to debug as well?

Reply:
The same question about lines 98, 99.

Reply:
I thought preferable to leave some key INFO messages to indicate when new keys are being created. That said, I'm happy to make everything DEBUG if you prefer.

Reply:
This was only a question. I have no strong opinion about what should be DEBUG in this code and what should not. I trust your decision.